Library/Homebrew/cmd/audit.rb

```diff
@@ -870,19 +870,19 @@ def audit_urls
 
       case p
       when %r[^http://ftp\.gnu\.org/]
-        problem "ftp.gnu.org mirrors should be https://, not http:// (mirror is #{p})."
+        problem "Please use https:// for #{p}"
       when %r[^http://[^/]*\.apache\.org/]
-        problem "Apache urls should be https://, not http (url is #{p})."
+        problem "Please use https:// for #{p}"
       when %r[^http://code\.google\.com/]
-        problem "code.google.com urls should be https://, not http (url is #{p})."
+        problem "Please use https:// for #{p}"
       when %r[^http://fossies\.org/]
-        problem "Fossies urls should be https://, not http (url is #{p})."
+        problem "Please use https:// for #{p}"
       when %r[^http://mirrors\.kernel\.org/]
-        problem "mirrors.kernel urls should be https://, not http (url is #{p})."
+        problem "Please use https:// for #{p}"
       when %r[^http://([^/]*\.|)bintray\.com/]
-        problem "Bintray urls should be https://, not http (url is #{p})."
+        problem "Please use https:// for #{p}"
       when %r[^http://tools\.ietf\.org/]
-        problem "ietf urls should be https://, not http (url is #{p})."
+        problem "Please use https:// for #{p}"
```

Comment:
Yes, It could be. I'd left it so far because I thought it was easier to read vertically rather than horizontally:

```ruby
    urls.each do |p|
      # Skip the main url link, as it can't be made SSL/TLS yet.
      next if p =~ %r[/ftpmirror\.gnu\.org]

      case p
      when %r[^http://ftp\.gnu\.org/], %r[^http://[^/]*\.apache\.org/], %r[^http://code\.google\.com/], %r[^http://fossies\.org/],
           %r[^http://mirrors\.kernel\.org/], %r[^http://([^/]*\.|)bintray\.com/], %r[^http://tools\.ietf\.org/]
        problem "Please use https:// for #{p}"
      when %r[^http://search\.mcpan\.org/CPAN/(.*)]i
        problem "MetaCPAN url should be `https://cpan.metacpan.org/#{$1}` (url is #{p})."
      end
    end
```

It doesn't look quite as crowded as I thought it would though.